tests/pages/package-edit.js

```diff
@@ -14,6 +15,12 @@ import { hasClassBeginningWith } from './helpers';
 import Toast from './toast';
 import Datepicker from './datepicker';
 
+@interactor class PackageSelectionStatus {
```

Comment:
We should definitely use the same interactor as for the show test since it is the same component.

Reply:
Moved to separate file and adjusted unit tests